### DISCUSSION TITLE
How to make group::Scroll background transparent?

### DISCUSSION THREAD

**Discussion by kjlaw89** [fltk-rs/fltk-rs#1579] [Q&A] [answered]
Hey all, I've run into an issue when decorating a scroll component. When I add some widgets to it and get a scroll my scroll widget automatically gets a background color:

![image](https://github.com/user-attachments/assets/a27435b5-9908-435a-9ce7-9f0d903c99f8)

I've tried setting FrameType::NoBox, as well as FlatBox with a custom draw method, with `Color::from_rgba(0, 0, 0, 0)`, which produces all black:

![image](https://github.com/user-attachments/assets/ff20bce0-1b2a-4adb-98e7-c78e7382a823)

It still has a background color even when there are no widgets in it...

```rust
// Create an options flex to hold all of the options and display text
let mut scroll_group = Scroll::default()
	.with_size(1200, 500);
	//.with_type(ScrollType::None);

scroll_group.set_color(Color::from_rgba(0, 0, 0, 0));
scroll_group.set_frame(FrameType::FlatBox);
scroll_group.begin();
```

Any ideas on what I could be missing?

Thanks!

**Comment by MoAlyousef** [comment, marked as answer]
Fl_Scroll (similarly Fl_Table) are always opaque because otherwise scrolling would cause drawing artifacts. You could always use `app::background(255, 255, 255)`. It would set the default color for a Scroll but it won't be transparent.

**Reply by kjlaw89** [reply]
Got it, so no way to do a transparent scroll because it would essentially have to redraw the entire screen to clean up any artifacts generated, that would make sense... I'll just have to do a simpler widget for this functionality then.

Thanks for the quick response!
